backend/api/views/filters_utils.py

```diff
@@ -141,3 +143,15 @@ def filter_queryset(self, request, queryset, view):
             django_filters[f'metadata_filters__{f["key"]}__icontains'] = f["value"]
 
         return queryset.filter(**django_filters)
+
+
+class ArchiveFilter(BaseFilterBackend):
+    def filter_queryset(self, request, queryset, view):
+        # apply filters to queryset
+        if view.action == "list":
+            archived = to_bool(request.query_params.get("archived"))
+            if archived:
+                return queryset
```

Comment:
Why we are not filtering on `archived=True` here?

Reply:
```python
archived = request.query_params.get("archived")
if view.action == "list" and archived is not None:
    queryset = queryset.filter(archived=to_bool(archived))
return queryset
```

Reply:
Note that I am quite sure we don't need to use custom code and rely on a BooleanField.

Reply:
Adding it to the datamanager filterset
```python
class DataManagerFilter(FilterSet):
    ...
    archived = BooleanFilter(field_name="archived", ...)
```

Reply:
Good catch it simplify the filtering, I also added a test on listing archived asset and the archived field as part of the serializer in [`f879ca`](https://github.com/Substra/substra-backend/pull/521/commits/f879ca257ef6b577c2ecb7f052a18df17aa1a985)